docarray/typing/tensor/abstract_tensor.py

```diff
@@ -54,26 +116,23 @@ def _to_node_protobuf(self: T) -> 'NodeProto':
 
     @staticmethod
     def _parse_item(item: Any) -> Tuple[Optional[str], Optional[str]]:
+        def _is_jaxtyping_class(_cls: type) -> bool:
+            return inspect.isclass(_cls) and issubclass(_cls, JaxTypingDType)
```

Comment:
why do u need the `isclass` inspection? isn't this already assumed from the `issubclass`?

Reply:
no, subclass will fail if the first input is not  a class, annoying